### PR TITLE
Fixed #7060 -- Added a note about race conditions to the tutorial

### PR DESCRIPTION
**Pull request by raphaelm**
This is a proposal for a note in the tutorial. Feel free to suggest any changes.

### CODE REVIEW

**Comment by timgraham** [comment]
Please follow the commit message guidelines as noted in our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist) and wrap lines at 79 characters.

File: docs/intro/tutorial04.txt [hunk 1]
```diff
@@ -179,6 +179,18 @@ Now, go to ``/polls/1/`` in your browser and vote in the question. You should se
...
+.. note::
+    The above code for our view ``vote()`` does have a small problem. If first gets
+    the ``selected_choice`` object from the database, then computes the new value of
+    ``votes`` and then saves it back to the database. If two users of your Web site
+    try to vote at *exactly the same time*, this might go wrong: The same value,
+    let's say 42, will retrieved for ``votes``. Then, for both users the new value
+    of 43 is computed and saved, while 44 would be the accurate value.
+
+    This is called a *race condition*. Once you finished the tutorial and are familiar
+    with the basics of Django, you should inform yourself about :doc:`Database transactions </topics/db/transactions>`
```
Comment (timgraham) [inline docs/intro/tutorial04.txt]: I'd suggest:
```
with the basics of Django, read :ref:`avoiding-race-conditions-using-f` to learn 
how you can solve this issue.
```

**Comment by raphaelm** [comment]
I agree with your suggestion, updated it accordingly and fixed the commit message.

**Comment by timgraham** [comment]
merged in 3e9b5bfd9c3e16c968278f7d050f52739690eb29, thanks!

**Pull request closed without merging**